mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -967,16 +968,13 @@ public void onSaveInstanceState(Bundle savedInstanceState) {
     private void onNotificationSelected(Intent intent) {
         Log.d(TAG, "Notification was selected");
 
-        mPendingOpenNotifications = true;
-        openNotificationsPageIfNeeded();
-
-        if (intent.hasExtra(EXTRA_MESSAGE)) {
-            new AlertDialog.Builder(this)
-                    .setTitle(getString(R.string.dlg_notification_title))
-                    .setMessage(intent.getStringExtra(EXTRA_MESSAGE))
-                    .setPositiveButton(getString(android.R.string.ok), null)
-                    .show();
+        mPendingOpenedNotificationId = intent.getStringExtra(EXTRA_NOTIFICATION_ID);
```

Comment:
I get the following error when using myopenhab:
````
05-23 15:23:07.976 12279-12279/org.openhab.habdroid D/OpenHABMainActivity: Notification was selected
05-23 15:23:07.977 12279-12279/org.openhab.habdroid W/Bundle: Key notificationId expected String but value was a java.lang.Integer.  The default value <null> was returned.
05-23 15:23:08.005 12279-12279/org.openhab.habdroid W/Bundle: Attempt to cast generated internal exception:
    java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
        at android.os.BaseBundle.getString(BaseBundle.java:1121)
        at android.content.Intent.getStringExtra(Intent.java:7139)
        at org.openhab.habdroid.ui.OpenHABMainActivity.onNotificationSelected(OpenHABMainActivity.java:971)
        at org.openhab.habdroid.ui.OpenHABMainActivity.processIntent(OpenHABMainActivity.java:377)
        at org.openhab.habdroid.ui.OpenHABMainActivity.onCreate(OpenHABMainActivity.java:269)
        at android.app.Activity.performCreate(Activity.java:7009)
        at android.app.Activity.performCreate(Activity.java:7000)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2731)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
````
The notification list opens, but nothing is highlighted.

Reply:
Hmm, this probably is the result of the same extra name being used in line 113 and 115 here: https://github.com/openhab/openhab-android/pull/848/files#diff-4f8ecc75b9dbc55f0cf179f386d56722R113 ... I wonder why this didn't happen for me (highlighting works), but different Android versions maybe handle this differently. I'll rename one of the extras.